src/client/index.ts

```diff
@@ -466,9 +466,7 @@ export class Client {
 		}
 
 		if (this.config.autoEnableTokens && Object.keys(this.tokenStore.getSelectedTokens()).length)
-			this.eventSender('tokens-selected', {
-				selectedTokens: this.tokenStore.getSelectedTokens(),
-			})
+			this.eventSender('tokens-selected', this.tokenStore.getSelectedTokens())
```

Comment:
@ I think it's still a good idea to keep the tokens nested, since it allows us to add other fields in the future without a breaking change. How about the event data looking like this: 
```
{
  tokens: 
  // We could add these later without breaking change
  tokenCount: number
  collectionCount: number
}
```

Reply:
Also please update eventSender tokens-selected interface with the new format.